src/client/app/home/home.component.html

```diff
@@ -1,6 +1,4 @@
-<p>
-  Howdy! Here's a list of awesome computer scientists. Do you know any others? Add to the list yourself.
-</p>
+<p>Howdy! Here's a list of awesome computer scientists. Do you know any others? Add to the list yourself.</p>
```

Comment:
Yes, totally, but it seems like pluralization is not working well (especially 'other'). [plnkr](http://plnkr.co/edit/SxWt8hnnCaKNy5AbXjxI?p=preview)